src/corner_kick/src/containers/Canvas/__helpers__/shapes.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * This file contains helper functions for unit testing
```

Comment:
should this be in the `__tests__` folder?

Reply:
Yes

Reply:
Moved